src/SFML/System/Angle.cpp

```diff
@@ -0,0 +1,38 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2022 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/System/Angle.hpp>
+
+
+namespace sf
+{
+////////////////////////////////////////////////////////////
+// Static member data
+////////////////////////////////////////////////////////////
+const Angle Angle::Zero;
```

Comment:
Shouldn't this be part of the class itself?
See the current implementation of `sf::Time::Zero`: https://github.com/SFML/SFML/blob/master/include/SFML/System/Time.hpp#L85
And the potential constexpr change for color: https://github.com/SFML/SFML/pull/1937#issuecomment-1022991504

Reply:
It is a member of the class, hence the `Angle::` namespace. See https://github.com/ChrisThrasher/SFML/blob/sf-angle/include/SFML/System/Angle.hpp#L96